packages/cli/src/rpc/explain/index/appmap-index.ts

```diff
@@ -126,7 +126,9 @@ export async function readAppMapContent(appmapFile: string): Promise<string> {
   appmapWords.push(...parameters);
   appmapWords.push(...types);
 
-  return appmapWords.join(' ');
+  // Words are separated by new lines to reduce the time spent tokenizing the content.
+  // Long lines may be slower to tokenize or be skipped altogether.
+  return appmapWords.join('\n');
```

Comment:
My assessment here is that changing this whitespace is inconsequential, except in the scenario described in the comment.